dependencies.yaml

```diff
@@ -77,9 +83,12 @@ dependencies:
     common:
       - output_types: [conda, pyproject, requirements]
         packages:
-          - sphinx
-          - pydata-sphinx-theme
           - myst-parser
+          - pydata-sphinx-theme
+          - sphinx
```

Comment:
These are just showing up in the diff because I wanted this list to be alphabetized.